### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

The `if ! wget -q --spider https://www.google.com/; then ...` check could hang under certain bad connection conditions. We already had a timeout for the git pull, so remove the general internet connectivity check and print messages to tell the user whether git pull timed out or not

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by rnzenha-s — APPROVED** [review]

**Review by niko-holmes — APPROVED** [review]

File: ansible/roles/products/common/hand_config/tasks/main.yml [hunk 1]
```diff
@@ -40,14 +35,30 @@
...
+          # Colours for echo
+          ORANGE='\033[0;33m'
+          GREEN='\033[0;32m'
+          NC='\033[0m' # No Color
+          timeout 5s git pull
+          retval=$?
+          if [ $retval -eq 124 ]; then
+            echo -e "${ORANGE}WARNING: git pull timed out after five seconds, NOT pulling latest hand calibration${NC}"
+          elif [ $retval -eq 1 ]; then
+            echo -e "${GREEN}git pull did not timeout${NC}"
```
Comment (toliver) [inline ansible/roles/products/common/hand_config/tasks/main.yml:48]: Maybe it would be clearer if we say:
"git pull finished successfully"

Comment (toliver) [reply ansible/roles/products/common/hand_config/tasks/main.yml:48]: Otherwise it may sound like it was expected to time out and it didn't.

Comment (carebare47) [reply ansible/roles/products/common/hand_config/tasks/main.yml:48]: I did have that originally, but it turns out that `git pull` can fail in a few different ways but still return `1`

Comment (carebare47) [reply ansible/roles/products/common/hand_config/tasks/main.yml:48]: oh no, my bad

Comment (carebare47) [reply ansible/roles/products/common/hand_config/tasks/main.yml]: nice, thanks
c3c10915a570e56bb31e1f79a70da55554a316af

**Review by toliver — APPROVED** [review]

**Pull request merged**